lib/packages/docutils/rstast.nim

```diff
@@ -316,3 +320,13 @@ proc renderRstToJson*(node: PRstNode): string =
   ##     "sons":optional node array
   ##   }
   renderRstToJsonNode(node).pretty
+
+proc dumpRstTree*(node: PRstNode, indent=0) =
```

Comment:
procs that echo are almost never the right thing to do, even for debugging. Instead, return a string and let test code use echo if they want to

Reply:
thanks, changed to returning a string.